poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.2 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
```

Comment:
Did someone do a `poertry lock` without the `--no-update` flag? Or are all of these transient dependencies of the few packages we did update?

Reply:
🤚